src/js/compat.ts

```diff
@@ -8,10 +8,14 @@ export const IN_NODE =
   typeof process.browser ===
     "undefined"; /* This last condition checks if we run the browser shim of process */
 
+export const IN_DENO = "Deno" in window;
```

Comment:
Some people have requested a `loadPyodide` "runtime" argument that can override the automatic feature detection. It might be a bit annoying to set up though.